manifests/base/cuda-rstudio-buildconfig.yaml

```diff
@@ -71,10 +71,11 @@ spec:
   resources:
     limits:
       cpu: "1"
-      memory: 8Gi
+      memory: 4Gi
     requests:
       cpu: "1"
-      memory: 8Gi
+      memory: 4Gi
+      ephemeral-storage: 50Gi
```

Comment:
I'm not a fan of the approach in https://github.com/red-hat-data-services/notebooks/pull/148#discussion_r1490214482

It seems sensible to me to just request the amount of ephemeral storage needed. According to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage this seems legit.

The problem with this is that over time this request would need to be kept up to date as the Dockerfile evolves, so that's another unpleasant and errorprone chore.